great_expectations/dataset/base.py

```diff
@@ -377,8 +327,8 @@ def _copy_and_clean_up_expectation(self,
                 # discards["result_format"] += 1
 
         if discard_include_configs_kwargs:
-            if "include_configs" in new_expectation["kwargs"]:
-                del new_expectation["kwargs"]["include_configs"]
+            if "include_config" in new_expectation["kwargs"]:
```

Comment:
Did this change cascade elsewhere?

Reply:
Line 67 names the kwarg `include_config`, not `include_configs`. This is consistent with lines 92, 616, and all other mentions of this keyword argument. All the implemented expectations also use the kwarg `include_config`, not `include_configs`. 

The old code that I deleted and replaced seemed to be an anomaly in that it used `include_configs`, so I changed it to make it consistent with everything else. The change shouldn't effect code elsewhere; it only makes the if statement meaningful since `if "include_configs" in new_expectation["kwargs"]` would always be false. 

`include_configs` still appears in the comments.  The kwarg `discard_include_configs_kwargs` in many of the config handling functions also makes things confusing since it is `discard_include_configs_kwargs` instead of  `discard_include_config_kwargs`. I think it might be worthwhile for me to go through the code and make everything either `include_config` or `include_configs` for the sake of consistency.

Reply:
rgr